content/github/managing-your-work-on-github/linking-a-repository-to-a-project-board.md

```diff
@@ -7,6 +7,11 @@ versions:
   free-pro-team: '*'
   enterprise-server: '*'
 ---
+{% note %}
+
+**Note:** In order to link a repository to your organization or user owned project board the repository needs to have Issues enabled. Please check out [this link](/github/managing-your-work-on-github/disabling-issues) to learn how to verify if Issues are enabled.
```

Comment:
> Since this link specifically refers to a sub-topic called "Disabling Issues", I added it within an extra sentence to avoid confusion.

This is a great observation. You're absolutely right to point out that a simple link that topic would have been confusing for users. ✨ 

I'm going to suggest an alternative fix because we actively avoid using this style of link (if you're interested in why, see [Make link text meaningful](https://readabilityguidelines.co.uk/content-design/links/#1-make-link-text-meaningful)). We also avoid mid-sentence links for accessibility and readability (see the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#links).

```suggestion
**Note:** In order to link a repository to your organization or user owned project board the repository needs to have issues enabled. For information on how to see if issues are disabled for a repository, see "[Disabling issues](/github/managing-your-work-on-github/disabling-issues) ."
```

Reply:
Thanks for the reference!

Reply:
No trouble